library/src/main/java/org/apache/apex/malhar/lib/state/managed/ManagedStateImpl.java

```diff
@@ -88,7 +88,7 @@ public void endWindow()
 
   @Min(1)
   @Override
-  public int getNumBuckets()
+  public long getNumBuckets()
```

Comment:
The number derived from the id of the bucket and it's always long. The reason why we could use int is because we use array to store the buckets and the id is always growing with system time. But now the bucket id would be more random and it's not a good idea to store in array so I change the num of buckets to long